docs/cost-estimate/05-iops.md

```diff
@@ -0,0 +1,203 @@
+# IOPS cost estimation per node
+
+> [!Note] 
+> This analysis compares IOPS (Input/Output Operations Per Second) requirements between 
+> Ouroboros Praos and Leios consensus protocols. IOPS is a critical metric that 
+> directly impacts storage performance and cloud costs.
+>
+> **Assumption on Votes and Certifications**: In the Leios protocol, votes are ephemeral and not
+> persisted to disk. Votes are used to certify EBs, and these certifications are then
+> included in RBs. The certifications become part of the permanent blockchain record
+> once included in an RB.
+
+## Ouroboros Praos
+
+In Ouroboros Praos, IOPS requirements are primarily driven by block validation 
+and ledger state updates.
+
+### Block Storage Operations
+
+| Operation Type | IOPS per Block | Notes |
+| -------------- | -------------- | ----- |
+| Block write    | 1              | Writing a validated block to disk |
+| Block read     | 0.1            | Reading blocks for chain verification (10% read rate) |
+| State update   | 3              | Ledger state updates per block |
+| **Total**      | **4.1**        | IOPS per block |
```

Comment:
My understanding is that IOPS are computed as writes/reads of ***filesystem blocks***, not logical operations like writing/reading a Praos/Leios block. AWS EBS seems to have a 4 kiB block size for the volumes that a cardano node would use.

So, to compute IOPS costs, we need to convert from Praos/Leios reads/writes to filesystem blocks.